--- ./fs/susfs.c
+++ ./fs/susfs.c
@@ -715,7 +715,7 @@
 	spin_unlock(&susfs_spin_lock);
 
		SUSFS_LOGI("fake_cmdline_or_bootconfig is set, length of string: %lu\n", strlen(fake_cmdline_or_bootconfig));
#else
-		SUSFS_LOGI("fake_cmdline_or_bootconfig is set, length of string: %u\n", strlen(fake_cmdline_or_bootconfig));
+		SUSFS_LOGI("fake_cmdline_or_bootconfig is set, length of string: %lu\n", strlen(fake_cmdline_or_bootconfig));
#endif
 		return 0;
	}
	SUSFS_LOGI("failed setting fake_cmdline_or_bootconfig\n");